app/views/layouts/base.html.erb

```diff
@@ -54,6 +54,7 @@
 </head>
 
 <body class="<%= Rails.env %>-env site <%= yield(:background_cls) %>">
+  <%= content_for?(:mobile_nav) ? yield(:mobile_nav) : yield %>
```

Comment:
Based on the build failures, I think the "else" condition here is causing the page content to be double-printed.

```suggestion
  <%= yield(:mobile_nav) if content_for?(:mobile_nav) %>
```